lib/fauna/client.rb

```diff
@@ -104,10 +104,23 @@ def delete(path, data = {})
     # for information on constructing queries.
     #
     # +expression+:: A query expression
+    # +expr_block+:: May be provided instead of expression. Block is used to build an expression with Fauna.query.
+    #
+    # Example using expression:
+    #
+    # <code>client.query(Fauna::Query.add(1, 2, Fauna::Query.subtract(3, 2)))</code>
+    #
+    # Example using block:
+    #
+    # <code>client.query { add(1, 2, subtract(3, 2)) }</code>
     #
     # :category: Query Methods
-    def query(expression)
-      post('', expression)
+    def query(expression = nil, &expr_block)
```

Comment:
It's an instance method on client, so I'm not sure if moving it makes sense. Is rdoc doing the wrong thing here? If it were moved up, would it be more prominent in rdoc?